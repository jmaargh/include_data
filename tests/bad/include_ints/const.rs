use include_data::include_ints;

fn main() {}

// Can't use for `const`
const BYTES_32_const: &[u8] = include_ints!(u8, "../../test_data/binary_32");
