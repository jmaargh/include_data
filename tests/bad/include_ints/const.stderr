error[E0013]: constants cannot refer to statics
 --> tests/bad/include_ints/const.rs:6:31
  |
6 | const BYTES_32_const: &[u8] = include_ints!(u8, "../../test_data/binary_32");
  |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: consider extracting the value of the `static` to a `const`, and referring to that
  = note: this error originates in the macro `include_ints` (in Nightly builds, run with -Z macro-backtrace for more info)
